<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserProfileWidget</class>
 <widget class="QWidget" name="UserProfileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UserProfileWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#name{ font-size:22px;color:#111111; font-weight: bold;}
QLabel#id{ font-size:12px;color:#333333;}
QLabel#dep{ font-size:12px;color:#333333;}
QLabel#btnAvatar{border:2px solid #A0FFFFFF;}
QLabel#btnAvatar:hover{border:2px solid #1194a4;}

QPushButton#btnBasic,#btnMore{height:27px;width:70px;color: #666666; background-color: #FFFFFF; border:none;}
QPushButton#btnBasic:hover,#btnMore:hover {color: #FFFFFF;background-color: #3dbbca;}
QPushButton#btnBasic:pressed,#btnMore:pressed{color: #FFFFFF; background-color: #1194a4;}
QPushButton#btnBasic:checked,#btnMore:checked{color: #FFFFFF; background-color: #1ba9ba;}

QPushButton#btnBasic,#btnMore{ border-left:1px solid #1ba9ba; border-top:1px solid #1ba9ba; border-bottom:1px solid #1ba9ba;}
QPushButton#btnBasic{ border-right:1px solid #1ba9ba;}


QWidget#infopanel{border-radius: 10px;background-color: #E2E2E2;margin: 20px 10px 20px 10px}


QLineEdit{ background-color: #00000000; border:none;padding-left:-2px;}
QLineEdit:hover{ border:1px solid #999999;}
QLineEdit:focus{ background-color: #FFFFFF; border-width: 1px; border-style: solid; border-color: #1BA9BA;}
QLineEdit:disabled{ background-color: #ECECEC; border:none;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="shadowbg" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QWidget" name="shadowinnerbg2" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="TitlebarWidget" name="titlebar" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="baseinfo" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <widget class="QLabel" name="btnAvatar">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>100</width>
              <height>100</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>更换头像</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize" stdset="0">
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
            <property name="flat" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="id">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>61</y>
              <width>341</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>wz.wang</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
           <widget class="QLabel" name="dep">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>85</y>
              <width>231</width>
              <height>61</height>
             </rect>
            </property>
            <property name="text">
             <string>QunarStaff/旅游度假事业部/技术/当地人客户端开发/</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>31</y>
              <width>331</width>
              <height>28</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="name">
               <property name="text">
                <string>王征wz</string>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sex">
               <property name="minimumSize">
                <size>
                 <width>27</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>27</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>219</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <widget class="QStackedWidget" name="detailinfos">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>532</width>
              <height>255</height>
             </rect>
            </property>
            <widget class="QWidget" name="basepage">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="infopanel" native="true">
                <widget class="QLabel" name="lbBoom">
                 <property name="geometry">
                  <rect>
                   <x>60</x>
                   <y>70</y>
                   <width>70</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>个性签名：</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
                <widget class="QLabel" name="lbLeader">
                 <property name="geometry">
                  <rect>
                   <x>60</x>
                   <y>100</y>
                   <width>70</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>直属领导：</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
                <widget class="QLabel" name="lbMobile">
                 <property name="geometry">
                  <rect>
                   <x>60</x>
                   <y>130</y>
                   <width>70</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>手机号：</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
                <widget class="QLabel" name="lbMark">
                 <property name="geometry">
                  <rect>
                   <x>60</x>
                   <y>160</y>
                   <width>70</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>备注：</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
                <widget class="QLineEdit" name="leBoom">
                 <property name="geometry">
                  <rect>
                   <x>140</x>
                   <y>70</y>
                   <width>361</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="placeholderText">
                  <string>点击编辑个性签名</string>
                 </property>
                </widget>
                <widget class="QLabel" name="lbLeaderLink">
                 <property name="geometry">
                  <rect>
                   <x>140</x>
                   <y>100</y>
                   <width>361</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
                 </property>
                </widget>
                <widget class="QLabel" name="lbMobileLink">
                 <property name="geometry">
                  <rect>
                   <x>140</x>
                   <y>130</y>
                   <width>361</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;点击查看&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="openExternalLinks">
                  <bool>false</bool>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
                 </property>
                </widget>
                <widget class="QLineEdit" name="leMark">
                 <property name="geometry">
                  <rect>
                   <x>140</x>
                   <y>160</y>
                   <width>361</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="placeholderText">
                  <string>点击编辑备注</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="leEmpty">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>0</width>
                   <height>0</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="morepage"/>
           </widget>
           <widget class="QWidget" name="infonav" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>532</width>
              <height>40</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnBasic">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>资料</string>
               </property>
               <property name="icon">
                <iconset resource="localman.qrc">
                 <normaloff>:/Images/aio_startsession_down.png</normaloff>:/Images/aio_startsession_down.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
